### DISCUSSION TITLE
Internal service page

### DISCUSSION THREAD

**Discussion by martinwake** [dwp/design-system#457] [General]
Hello, we'd like to use this open discussion to talk about the [Internal service page template](https://design-system.dwp.gov.uk/patterns/internal-service-page) in the DWP design system.

This page template uses the [internal service header](https://design-system.dwp.gov.uk/components/internal-service-header) and [internal service footer](https://design-system.dwp.gov.uk/components/internal-service-footer) to provide a basic page for use in internal DWP services.

Please use this discussion to share any research, insights, experiences or problems with this pattern that you think might be useful for others.

**Comment by timcooper-DWP** [comment]
The government prototype includes a favicon (and maybe touch pngs too?)... should we incorporate the DWP logo for internal services?
